tests/ecdhp256-test.c

```diff
@@ -31,6 +35,13 @@ int main()
 {
 	uint8_t* result = (uint8_t*) malloc (sizeof (uint8_t) * 64);
 	bool ok = true;
+	uint8_t* pk = (uint8_t*) malloc (sizeof (uint8_t) * 64);
+	uint8_t* decompressedPoint = (uint8_t*) malloc (sizeof (uint8_t) * 64);
+	uint64_t incorrect = 18446744073709551615U;
+	uint64_t correct = 0;
+
+// The start of NIST ECDH Initiator tests
```

Comment:
can you separate each flavor of tests into its own function? e.g.
```
bool test_nist() {
}

bool test_compression () {
}

...

int main () {
  if (!test_compression())
    return EXIT_FAILURE;
  if (!test_nist())
    return EXIT_FAILURE;
  return EXIT_SUCCESS;
}
```

or something like that
```